### PR TITLE
responsive-input

### PR DESCRIPTION
**Pull request by pPitohu**
created customized input based on mui textfield, added Montserrat font

adding font using html <link> tag, because it's throwing error implementing with 'import' in tsx file.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -60,7 +79,7 @@
         "import/no-unresolved": 1,
         "no-shadow": "off",
         "linebreak-style": "off",
-        "no-unused-vars": "warn",
+        "no-unused-vars": "off",
```
Comment (psychobug420) [inline .eslintrc.json]: No unused vars should be error or warn, you can off it in several files by eslint directives

File: src/UI/StyledTextInput.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                borderWidth: '2px',
+            },
+            '&:after': {
+                borderColor: theme.palette.primary.dark,
+            },
+        },
+    },
+};
+
+const StyledTextInput = ({
```
Comment (psychobug420) [inline src/UI/StyledTextInput.tsx]: Rename StyledTextInput to TextInput, we will add styled components later and Styled prefix reserved for them

File: src/UI/StyledTextInput.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        size={size ?? 'small'}
+        variant={variant ?? 'outlined'}
+        color={color}
+        className={classes.input}
+        type={type}
+        placeholder={placeholder}
+        helperText={helperText}
+    />
+);
+export default withStyles(styles)(StyledTextInput);
```
Comment (psychobug420) [inline src/UI/StyledTextInput.tsx]: Okay, this project would have default exports, but I don't like them and have several reasons for it

File: src/UI/StyledTextInput.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    helperText,
+}: WithStyles<typeof styles> & TextFieldProps) => (
+    <TextField
+        size={size ?? 'small'}
+        variant={variant ?? 'outlined'}
+        color={color}
+        className={classes.input}
+        type={type}
+        placeholder={placeholder}
+        helperText={helperText}
```
Comment (psychobug420) [inline src/UI/StyledTextInput.tsx]: Add extra props with spread operator {...extra} for this component

File: src/App.tsx [hunk 5]
```diff
@@ -1,10 +1,16 @@
...
+import React from 'react';
+import StyledTextInput from './UI/StyledTextInput';
 
 const App: React.FC = () => (
     <div className="App">
-        EAZY LEARN
-        <Test />
+        <StyledTextInput
+            placeholder="Логин"
+            type="text"
```
Comment (psychobug420) [inline src/App.tsx]: Why type text if we use text input???

File: src/App.tsx [hunk 6]
```diff
@@ -1,10 +1,16 @@
...
+            color="primary"
```
Comment (psychobug420) [inline src/App.tsx]: Color should be primary by default

File: src/App.tsx (hunk 5 above)
Comment (pPitohu) [reply src/App.tsx]: this type is default, why not to use it?

**Review by psychobug420 — APPROVED** [review]

**Pull request merged**